pkgs/development/libraries/cassandra-cpp/default.nix

```diff
@@ -0,0 +1,34 @@
+{ pkgs, examples ? false}: pkgs.stdenv.mkDerivation rec {
+    pname = "cassandra-cpp";
+    version = "2.16.2";
+
+    src = pkgs.fetchgit {
```

Comment:
Any reason not to use fetchFromGitHub ? (it is more bandwidth efficient)

Reply:
No one. Fixed!